### DISCUSSION TITLE
Let's say I export my yolo model to ONNX format, can I still use sahi for slicing? How?

### DISCUSSION THREAD

**Discussion by yrik** [obss/sahi#890] [Q&A]
I feel like I would need to decouple the logic that is responsible for slicing, applying prediction and combining predictions into the final one. Any hints where to start?

**Comment by liquidweb9** [comment]
Have you resolved the issue now? I'm experiencing slow detection speeds when using YOLOv5. I'm hoping to accelerate the detection using ONNX, or do you have any other alternative solutions?
